ui/css/_sitewide.scss

```diff
@@ -0,0 +1,157 @@
+body {
+	background: #ede0cd;
+}
+
+body.video {
+	background: #211913;
+}
+
+a {
+	color: #625044;
+}
+
+.footer-wrapper {
+	background: #ffffff;
+}
+
+.footer {
+	color: #211913;
+}
+
+.footer p {
+	text-align: left;
+}
+
+.content-wrapper h1,
+h1 {
+	background: none;
+	color: #211913;
+	font-family: $font-header;
+	font-size: 45px;
+	font-weight: 700;
+	line-height: 1.5em;
+	margin-bottom: 0;
+	padding-top: 60px;
+	padding-bottom: 60px;
+	text-align: center;
+	text-transform: uppercase;
+}
+
+body.video h1 {
+	color: #ede0cd;
+}
+
+.content-wrapper h2,
+h2 {
+	color: #211913;
+	font-family: 'Oswald', sans-serif;
```

Comment:
Should all font names, colors, etc be replaced with SASS vars?

Reply:
Yes, just missed that one.